### PR TITLE
[HEBAO_V1] add unit test for LoopringModule

### PR DESCRIPTION
**Pull request by kongliangzhong**

### CODE REVIEW

File: packages/hebao_v1/contracts/test/DummyExchange.sol [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+    function deposit(
+        address tokenAddress,
+        uint96  amount
+        )
+        external
+        payable
+        override
+    {
+
```
Comment (letsgoustc) [inline packages/hebao_v1/contracts/test/DummyExchange.sol]: Looks we should emit Deposit event.

File: packages/hebao_v1/contracts/test/DummyExchange.sol [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    function withdraw(
+        address tokenAddress,
+        uint96  amount
+        )
+        external
+        payable
+        override
+    {
+
```
Comment (letsgoustc) [inline packages/hebao_v1/contracts/test/DummyExchange.sol]: Looks we should emit Withdraw event.

File: packages/hebao_v1/test/testLoopringModule.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            []
+          ),
+          ctx,
+          true,
+          wallet,
+          [owner],
+          { from: owner }
+        );
+
+        await assertEventEmitted(loopringModule, "AccountUpdated");
```
Comment (adamchen0510) [inline packages/hebao_v1/test/testLoopringModule.ts]: We should check the msg.sender in AccountCreated(emit in exchange) is wallet

File: packages/hebao_v1/contracts/test/DummyExchange.sol (hunk 2 above)
Comment (kongliangzhong) [reply packages/hebao_v1/contracts/test/DummyExchange.sol]: All the events I checked in test/testLoopringModule.ts were emitted in LoopringModule.sol.
So I think I will just remove all the events in DummyExchange.sol

File: packages/hebao_v1/test/testLoopringModule.ts (hunk 3 above)
Comment (kongliangzhong) [reply packages/hebao_v1/test/testLoopringModule.ts]: Updated. please take another look.

**Pull request merged**